default_app/main.js

```diff
@@ -252,37 +255,26 @@ function loadApplicationPackage (packagePath) {
       app.setPath('userCache', path.join(app.getPath('cache'), app.getName()))
       app.setAppPath(packagePath)
     }
-    const Module = require('module')
+
     try {
       Module._resolveFilename(packagePath, module, true)
     } catch (e) {
-      showErrorMessage('Unable to find Electron app.\n\n' +
-        `See: ${packagePath}`)
+      showErrorMessage(`Unable to find Electron app at ${packagePath}.\n\n${e.message}`)
+      return
     }
+
     // Run the app.
     Module._load(packagePath, module, true)
   } catch (e) {
-    if (e.code === 'MODULE_NOT_FOUND') {
```

Comment:
This is now handled by the `Module._resolveFilename` call a few lines above so I can think we can safely remove it from here.